pkg/setting/setting.go

```diff
@@ -1072,9 +1078,37 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 		ConnStr: connStr,
 	}
 
+	// Annotations
+	dashboardAnnotation := iniFile.Section("annotations.dashboard")
+	APIAnnotation := iniFile.Section("annotations.api")
```

Comment:
Local variable naming should be camelCase.
```suggestion
	apiIAnnotation := iniFile.Section("annotations.api")
```